js/main.js

```diff
@@ -20,8 +20,102 @@ function handleSubmit(event) {
   data.nextEntryId++;
   data.entries.unshift(entry);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  var instantEntry = entryTemplate(data.entries[0]);
+  $ul.prepend(instantEntry);
   $form.reset();
+  switchView($entriesPage.getAttribute('data-view'));
+  data.view = 'entries';
 }
 
 var $form = document.querySelector('form');
 $form.addEventListener('submit', handleSubmit);
+
+function entryTemplate(entry) {
+  var $entryContainer = document.createElement('li');
+  var $row = document.createElement('div');
+
+  $row.setAttribute('class', 'row');
+
+  var $columnImage = document.createElement('div');
+  $columnImage.setAttribute('class', 'column-half');
+
+  var $userImage = document.createElement('img');
+  $userImage.setAttribute('src', entry.photoUrl);
+  $userImage.setAttribute('class', 'width-100');
+  $userImage.setAttribute('alt', 'user-image');
+
+  var $columnContent = document.createElement('div');
+  $columnContent.setAttribute('class', 'column-half');
+
+  var $heading = document.createElement('h2');
+  $heading.textContent = entry.title;
+  var $note = document.createElement('p');
+  $note.textContent = entry.note;
+
+  $entryContainer.appendChild($row);
+  $row.appendChild($columnImage);
+  $columnImage.appendChild($userImage);
+  $row.appendChild($columnContent);
+  $columnContent.appendChild($heading);
+  $columnContent.appendChild($note);
+
+  return $entryContainer;
+}
+
+/*
+<li>
+  <div class="row">
+    <div class="column-half">
+      <img
+        src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/45/A_small_cup_of_coffee.JPG/1200px-A_small_cup_of_coffee.JPG"
+        alt="user-image" class="width-100">
+    </div>
+    <div class="column-half">
+      <h2>Coffee</h2>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione dicta voluptates reprehenderit
+        alias
+        nobis atque aperiam! Nobis cumque amet molestiae dicta voluptates reprehenderit alias
+        nobis atque aperiam!</p>
+    </div>
+  </div>
+</li>
+*/
```

Comment:
I would put this inside your `entryTemplate` function so it's clear that it is the structure that function is creating.